aws_org_tools/github_oidc/create_github_action_identity_providers.py

```diff
@@ -0,0 +1,209 @@
+import json
+import sys
+from pathlib import Path
+from urllib.parse import urlparse
+
+from github_action_config import (
+    GITHUB_ACTIONS_PERMISSIONS,
+)
+
+# Add the parent directory to the path
+root_dir = Path(__file__).resolve().parents[2]
+sys.path.insert(0, str(root_dir))
+
+from aws_org_tools.utils import EachAccount  # noqa: E402
+
+
+@EachAccount()
+def configure_github_oicd_provider(account):
+    iam_client = account.session.client("iam")
+    account_name = account.name
+    account_id = account.account_id
+    account_details = f"'{account_name}' ({account_id})"
+    print(
+        f"\n{'=' * len(account_details)}\n{account_details}\n{'=' * len(account_details)}"
+    )
+    repos = GITHUB_ACTIONS_PERMISSIONS.get(account_name, None)
+    github_oidc_url = "https://token.actions.githubusercontent.com"
+
+    if repos is None:
+        print(
+            f"No entry for '{account_name}' found in GITHUB_ACTIONS_PERMISSIONS"
+        )
+        return
+
+    if len(repos) == 0:
+        print(
+            f"No repos found in GITHUB_ACTIONS_PERMISSIONS entry for '{account_name}'."
+        )
+        return
+
+    print(
+        f"Doing get or create of Github OIDC Provider '{account_name}' ({account_id})"
+    )
+    github_oidc_provider_arn = get_github_oidc_provider(
+        iam_client, account_id, github_oidc_url
+    )
+    if github_oidc_provider_arn:
+        print(f"    Found GitHub OIDC provider: {github_oidc_provider_arn}")
+    if not github_oidc_provider_arn:
```

Comment:
This looks like it could just be
`else:`

Reply:
[d996b15](https://github.com/DemocracyClub/dc-aws-org-tools/pull/1/commits/d996b150361013c5bae9d5b04abd2940ad5e0141)